src/React.re

```diff
@@ -886,6 +887,31 @@ external useDebugValue: ('value, ~format: 'value => string=?, unit) => unit =
 
 module Experimental = {
   /* This module is used to bind to APIs for future versions of React. There is no guarantee of backwards compatibility or stability. */
+  [@mel.module "react"] external usePromise: Js.Promise.t('a) => 'a = "use";
+  [@mel.module "react"] external useContext: Context.t('a) => 'a = "use";
+  [@mel.module "react"] external use: 'a => 'b = "use";
```

Comment:
should we default the name `use` to `Js.Promise.t('a)` and add `useAny` for `'a => 'b`?

Reply:
What's the use case for `use : 'a -> 'b`, which is essentially `Obj.magic`?